### ISSUE TITLE
Mouted disk and all content on it are visible to another logged users (with admin and without admin privileges)

### ISSUE THREAD

**Issue by ezyuzin** [bailey27/cppcryptfs#131]
Windows 10 Pro, 1909
build: 1.4.3.7 64bit
dokany: 1.5.0.3000

Steps for reproduce:
1. add new local user
2. Mount crypted folder as disk or mount to folder
3. switch to created user, locate disk or mounted folder
4. Use decrypted content as owner.

truecrypt/veracrypt have a same issue, described here
https://www.truecrypt71a.com/documentation/security-requirements-and-precautions/multi-user-environment/

encfsmp for win not have this issue, another logged users can see a disk or mounted folder, but cannot open it

**Comment by bailey27** [comment]
Thanks for pointing this out.  I'm working on a solution.

**Comment by bailey27** [comment]
The way I've implemented this is that there will be an option that if turned on then if any user other than the user who started cppcryptfs attempts to access a mounted volume, then they will get "Access Denied" on every access.

Is this acceptable?

**Comment by ezyuzin** [comment]
> Is this acceptable?
absolutely! 👍

**Comment by bailey27** [comment]
I've released 1.4.3.8 with a "Deny other users" setting.  I did my best to test this setting thoroughly and could not access a volume mounted by a different user when using this setting.

Though I think mounted volumes are safe from access by a different user when using this setting, there does exist the possibility that  a clever and determined attacker could possibly could find a way to access it from another account.
